custom_components/tahoma/tahoma_entity.py

```diff
@@ -128,7 +128,7 @@ def device_info(self) -> Dict[str, Any]:
 
         return {
             "identifiers": {(DOMAIN, self.base_device_url)},
-            "name": self.name,
+            "name": self.device.label,
```

Comment:
If I don’t do this, HA will take the name of the first loaded entity of the device. In my case, the switch.